horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
@MarkusHackspacher I think we should continue thinking for ourselves and not blindly following other projects without any thought.

I'm confused by your example. There is additional whitespace, or am I going crazy?

```python
"grey_light"    :   pygame.Color(200, 200, 200),
```

And by the way, I didn't find any mention of either `pep8`, `pycodestyle`, `pyflakes` or `flake8` in that repo. Looking at the library code, I seriously doubt they do any style checks at all.
Perhaps that should inspire us.